README.md

```diff
@@ -1,7 +1,7 @@
 This module allows you to easily extend the built-in Error type for typed error checking
 
 ```coffee-script
-TypedError = require 'typed-error'
+{ TypedError } = require 'typed-error'
```

Comment:
You don't have to make this breaking change if you don't want to. No idea how much hassle such a change would be, or if you're just wanting to move to the ES6 modules syntax anyway, but if you don't want this you could just `export = TypedError`, and stick with the same external API as before.